### ISSUE TITLE
Please clarify project name

### ISSUE THREAD

**Issue by mardy** [nmoehrle/mvs-texturing#72]
Hi! Just found out about this program, tried it, and got very impressed by the result!
I'm making a snap package of it, so that it will be easy to install on Linux distributions (see https://snapcraft.io for more info, or talk with Simon from MVE about this, since I've been annoying him as well :-)).

I've one simple question about the project name: what's the official name? The website calls it TexRecon, while the project here in github says mvs-texturing. Which one is preferred?

I actually wonder about the relation between this project and MVE; would it be appropriate if I simply add texrecon in the MVE snap package? Or do you prefer them to stay separate?

**Comment by nmoehrle** [comment]
Good question. When I started the project I called it mvs-texturing to reflect the focus on MVS reconstructions. We later renamed the binary to `texrecon` following the convention our group uses for binary names (`sfmrecon`, `dmrecon`, `fssrrecon` and later [`smvsrecon`](https://github.com/flanggut/smvs)). I did not want to rename the repository because it would have cut ties to other forks, existing build scripts etc.

I still refer to the project as mvs-texturing but the author of the website changed the name during a recent update. I'll start a discussion with him tomorrow and I hope to clarify this for the future.

With respect to the snap package I would use mvs-texturing since it corresponds to the repo name and maybe change it later together with the repo name.

There are two reasons why MVE and this project are separate: maintainer and dependencies.
Simon is still maintaining MVE and I will continue to maintain this project in the foreseeable future so I would prefer two separate packages as well. This also makes sense w.r.t. dependencies, we would like to keep the dependencies of MVE to a minimum (like most frameworks).

The licensing of GCO is a little problematic and we should not distribute it. I am working to remove GCO entirely with mapmap in the [mapmap](https://github.com/nmoehrle/mvs-texturing/tree/mapmap) branch so it would make sense to use branch for the snap package.
